src/util/persistent_logger.js

```diff
@@ -198,7 +198,7 @@ class PersistentLogger {
 
             try {
                 // Finally replace the current active so as to consume them in the next iteration
-                await failure_log._replace_active();
+                await failure_log._replace_active(!result);
```

Comment:
We should log `ENOENT` if the last processed journal (aka log) in a journal namespace was not deleted?

Reply:
Yes, the not-delete file needs to be replaced. (That's what I understand, correct me if I'm wrong. In my use case I always delete the the processed files.)

Reply:
Actually, I think I missed something when I was looking at this last time. This makes sense.